src/ardllocc.c

```diff
@@ -1,42 +1,28 @@
 /** @file
-    @author ATOR @date 2014-12-04
```

Comment:
I would perfer if you left the author tag at all levels - file, subprogram, and module. Each are different (in general).

I realize in your code you have one subprogram per file, but it needed stay that way. You could will the lottery, move to Tahiti, and then your replacement might decide to merge many subprograms in one file. The file authorship will then be different.

In general, as software libraries like this are maintained over the years and decades, this author information will no longer seem so redundant. Code arrangement drifts over time...